cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -236,6 +236,7 @@ public static String getZKQuorum(CConfiguration cConf) {
     public static final String PROGRAM_MAX_STOP_SECONDS = "app.program.max.stop.seconds";
     public static final String YARN_ATTEMPT_FAILURES_VALIDITY_INTERVAL =
       "app.program.yarn.attempt.failures.validity.interval";
+    public static final String SUPPORT_BUNDLE_DIR = "app.support.bundle.dir";
```

Comment:
Please don't put it under App Fabric. It currently lives in App Fabric pod, but generally is fully independent